app/kafka/manager/utils/ZkUtils.scala

```diff
@@ -34,6 +34,8 @@ object ZkUtils {
   val BrokerTopicsPath = "/brokers/topics"
   val TopicConfigPath = "/config/topics"
   val TopicConfigChangesPath = "/config/changes"
+  val LogkafkaConfigPath = "/logkafka/config"
```

Comment:
Let's move all of these changes into it's own class LogKafkaZkUtils, as this class also attempts to replicate Kafka's ZkUtils class.